core/templates/dev/head/pages/exploration_player/conversation_skin_embed_directive.html

```diff
@@ -365,12 +365,17 @@
       bottom: -50px;
       position: absolute;
     }
+    .help-card-fixed {
```

Comment:
This introduces different type of issue.
when browsers viewport's height is small, help-card won't be visible.